src/core/qgismobileapp.cpp

```diff
@@ -957,7 +957,8 @@ bool QgisMobileapp::print( const QString &layoutName )
   if ( !layoutToPrint || layoutToPrint->pageCollection()->pageCount() == 0 )
     return false;
 
-  layoutToPrint->referenceMap()->zoomToExtent( mMapCanvas->mapSettings()->visibleExtent() );
+  if ( layoutToPrint->referenceMap() )
+    layoutToPrint->referenceMap()->zoomToExtent( mMapCanvas->mapSettings()->visibleExtent() );
   layoutToPrint->refresh();
```

Comment:
Do we need to refresh or can we take that into the if?

Reply:
To be safe, I think we should -- refresh will also refresh non-map layout items. Tables, HTML, expression-based labels, etc.